tests/test_shell.py

```diff
@@ -129,13 +150,35 @@ def test_index(self):
         doc = json.loads(self.call('python -m signac index'.split()))
         assert 'statepoint' in doc
         assert doc['statepoint'] == {'a': 0}
+        doc = json.loads(self.call('python -m signac project --index'.split()))
+        assert 'statepoint' in doc
+        assert doc['statepoint'] == {'a': 0}
         project.open_job({'a': 0}).document['b'] = 0
         doc = json.loads(self.call('python -m signac index'.split()))
         assert 'statepoint' in doc
         assert doc['statepoint'] == {'a': 0}
         assert 'b' in doc
         assert doc['b'] == 0
 
+    def test_document(self):
+        self.call('python -m signac init my_project'.split())
+        self.call('python -m signac project --access'.split())
+        project = signac.Project()
+        job_a = project.open_job({'a': 0})
+        job_a.init()
+        assert len(project) == 1
+        job_a.document['data'] = 4
+        doc = json.loads(self.call('python -m signac document'.split()))
+        assert 'data' in doc
+        assert doc['data'] == 4
+        doc = json.loads(self.call('python -m signac document {}'.format(job_a.id).split()))
+        assert 'data' in doc
+        assert doc['data'] == 4
+        out = self.call('python -m signac document --pretty'.split())
```

Comment:
Same issue as above with JSON compatibility and single quotes with `--pretty`, double quotes without `--pretty`.

Reply:
Same comment as above.